app/models/annotation/AnnotationService.scala

```diff
@@ -155,14 +155,19 @@ class AnnotationService @Inject()(
         elementClassToProto(
           fallbackLayer.map(layer => layer.elementClass).getOrElse(VolumeTracingDefaults.elementClass)),
         fallbackLayer.map(_.name),
-        fallbackLayer.map(_.largestSegmentId).getOrElse(VolumeTracingDefaults.largestSegmentId),
+        initialLargestSegmentId(fallbackLayer.map(_.largestSegmentId)),
         0,
         VolumeTracingDefaults.zoomLevel,
         organizationName = Some(organizationName),
         resolutions = resolutionsRestricted.map(vec3IntToProto)
       )
   }
 
+  private def initialLargestSegmentId(fromFallbackLayer: Option[Option[Long]]): Option[Long] =
+    if (fromFallbackLayer.isDefined) {
+      fromFallbackLayer.flatten
+    } else VolumeTracingDefaults.largestSegmentId
```

Comment:
This means the outer option is checked, right? So if there is a fallback layer, its largest segment id is used, even if its None, correct?